core/src/main/java/io/ddf/content/IHandleViews.java

```diff
@@ -24,12 +24,14 @@
    */
   public List<Object[]> getRandomSample(int numSamples, boolean withReplacement, int seed);
 
-  public DDF getRandomSampleByNum(int numSamples,
+  public DDF getRandomSampleByNum(long numSamples,
```

Comment:
then maybe let's change the function names? Those names sound a bit too verbose. Probably something like this:

- `getRandomSample` -> `sample(double fraction, boolean replacement, int seed)`
- `getRandomSampleByNum` -> `sample(long size, boolean replacement, int seed)`
- `getRandomSampleApprox` -> `sampleApprox(double fraction, boolean replacement, int seed)`